src/Harness/Program.cs

```diff
@@ -0,0 +1,30 @@
+using Dte.Common.Lambda;
+using Harness.Startup;
+
+var builder = WebApplication.CreateBuilder(args);
+var executionEnvironment = new LambdaExecutionEnvironment
+{
+    EnvironmentName = "Harness",
```

Comment:
Go with `Development` here.
.Net uses standard environment names for a few things (like DI) and it might be useful for them to turn dev functionality on when using the harness.